src/Notifier.cpp

```diff
@@ -52,8 +54,8 @@ bool Notifier::Initialize(const size_t handshakeCount) {
       gQueue.reset(new Shotgun);
       try {
          gQueue->Aim(GetNotifierQueueName());
-      } catch (std::exception& e) {
-         LOG(WARNING) << "Caught exception: " << e.what();
+      } catch (std::string& e) {
```

Comment:
forever as in Aim() always throws a string and has been since i've known the Shotgun class.